config/locales/en.yml

```diff
@@ -266,13 +266,10 @@ en:
       expired_payment: These payment sources were not valid at the time of fulfillment. Please choose a different payment source.
       download_unjournalable: Export Errors as CSV
       no_transactions: There are no transactions requiring journals matching the search parameters.
-    year_end_modal:
-      header: Year End Closing Reminder
-      message: |
-        We are in the year-end closing window.  Any journals for the previous fiscal year
-        must be submitted by %{cutoff_time}.  If you are creating a journal for %{year},
-        make sure to update the Journal Date field.
-      proceed: Proceed
+    journal_closing_modal:
+      header: Journal Closing Reminder
+      proceed: Create Journal
+      cancel: Edit Journal Date
```

Comment:
Fair point - the idea is that in order to edit the journal date you need to cancel (close) the modal.

Reply:
unless this action takes you back to editing the record where you can change the date, it might be better to say "Cancel & edit" or "Go Back and Edit Date"

Reply:
The button will hide the modal, revealing the form where the user can edit the Journal Date field.  I agree this is a little odd.  I would just go with "Continue"/"Cancel" myself, but the client specifically requested this text for the buttons.